lib/turbolinks.rb

```diff
@@ -7,19 +7,27 @@
 require 'turbolinks/redirection'
 
 module Turbolinks
+  module ControllerConcern
```

Comment:
Yeah, I like that distinction, Matthew.

Thibaut, looking forward to the readme :+1:

--
Kasper

> Den 04/05/2015 kl. 14.14 skrev Thibaut Courouble <notifications@github.com>:
> 
> In lib/turbolinks.rb:
> 
> > @@ -7,19 +7,27 @@
> >  require 'turbolinks/redirection'
> >  
> >  module Turbolinks
> > +  module ControllerConcern
> @matthewd @kaspth Thanks for the feedback. Updated.
> 
> —
> Reply to this email directly or view it on GitHub.
>